src/Rector/Contrib/PHPUnit/SpecificMethod/AssertTrueFalseInternalTypeToSpecificMethodRector.php

```diff
@@ -110,10 +119,8 @@ private function moveFunctionArgumentsUp(MethodCall $methodCallNode): void
         unset($oldArguments[0]);
 
         $methodCallNode->args = array_merge([
-            new Arg(new String_(
-                $this->oldMethodsToTypes[$isFunctionName]
-            )),
-            new Arg($argument),
+            $this->nodeFactory->createArg(new String_($this->oldMethodsToTypes[$isFunctionName])),
```

Comment:
@TomasVotruba As we have `createTrueConstant` and like methods in `NodeFactory`, could we have `createString`? :thinking: